### ISSUE TITLE
SwipeView inaccessible for screenreader users on Android

### ISSUE THREAD

**Issue by dawidpieper** [dotnet/maui#6648]
### Description

On Android there's an official context menu, that can be accessed from TalkBack or Commentary Screenreader. Unfortunately actions assigned with SwipeView are not included in it. Therefore blind users currently aren't able to access SwipeView options.

### Steps to Reproduce

1.. Create a SwipeView with any options, bind to any control.
2. It is impossible to access its options using assistive technologies.

### Version with bug

Release Candidate 2 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android, I was *not* able test on other platforms

### Affected platform versions

Android 11

### Did you find any workaround?

The workaround would be to define a custom button programatically opening SwipeView.
    However, since each item on the list would have to have such a button, the interface becomes inconvenient to use.

**Comment by dawidpieper** [comment]
I am happy to work on it when we come up with a reasonable solution.
Unfortunately, I have no idea on my part, apart from defining custom menu, opened with, for example, a long tap, containing all options invoked using all possible swipes.

**Comment by dawidpieper** [comment]
After 6.0.312 release I suppose it should be an a11y priority, as SwipeView on Android remains unusable for screenreader users.

**Comment by dawidpieper** [comment]
I have thought about this problem and see two possible solutions, although unfortunately neither is perfect.
1. Implementing, for accessibility purposes only, a native context menu with options from all swiping directions.
I can't say how difficult it would be to implement, I don't know the Android API so much.
2. Implementing a custom menu opened by double tapping.
This menu would include options from all swipes.

I have no better idea, but at the moment we have a control completely inaccessible to screen reader users.
As ```CollectionView``` is proposed over the old ```ListView``` and has no other context menu mechanism, I find this a huge accessibility problem.

CC @jfversluis @PureWeen

**Comment by kaiba78987** [comment]
Hi! I was looking for accessibility info about MAUI and I found this issue. As accessibility expert and blind user, I think the best way to deal with this is to use the accessibility contextual actions for each platform. This will keep the native experience and should work better with any assistive technology.
iOS: You should add all the actions in the custom action list. https://developer.apple.com/documentation/objectivec/nsobject/uiaccessibilityaction
Android: You can add every action to the local context menu or show a contextual modal with all the options by a long press on the element.
This is the approach applied by other frameworks such as React Native, although they just include the actions in one list.
I supose on other desktop platforms it would be as simply as showing the context menu on pressing the contextual menu key shortcut, as expected.

Thanks in advance for your interest in making accessible software.

**Comment by Zhanglirong-Winnie** [comment]
Verified this issue with Visual Studio Enterprise 17.8.0 Preview 2.0. Not repro on android platform.
